modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -648,6 +648,62 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
         static Ptr<Layer> create(const LayerParams& params);
     };
 
+    class CV_EXPORTS CopyLayer : public Layer
+    {
+        public:
+
+            String intype, outtype;
+            bool forceCopy, dontCast;
+
+            static Ptr<Layer> create(const LayerParams& params);
+    };
+
+    class CV_EXPORTS AddConstantLayer : public Layer
+    {
+        public:
+
+            float constant_scalar, inplace;
+
+            static Ptr<Layer> create(const LayerParams& params);
+    };
+
+    class CV_EXPORTS PixelShuffleLayer : public Layer
+    {
+        public:
+
+            float upscaleFactor;
+
+            static Ptr<Layer> create(const LayerParams& params);
+    };
+
+    class CV_EXPORTS ParallelTableLayer : public Layer
+    {
+        public:
+
+            static Ptr<Layer> create(const LayerParams& params);
+    };
```

Comment:
ParallelTable is not a layer but a way Torch splits the topology on branches. Maybe we just need to process it like all other `*Table` entries?

https://github.com/opencv/opencv/blob/ec43292e1ea9da963e67427505b4113750829c3e/modules/dnn/src/torch/torch_importer.cpp#L501

Reply:
I will try the above approach locally.